<ui version="4.0" >
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Settings</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <widget class="QTabWidget" name="w_tabs" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="general" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>598</width>
        <height>360</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>&amp;General</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_7" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>Language</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QComboBox" name="w_language" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>Weights</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QWidget" native="1" name="widget" >
         <property name="minimumSize" >
          <size>
           <width>24</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="w_metric" >
            <property name="text" >
             <string>Metric (g)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="w_imperial" >
            <property name="text" >
             <string>Imperial (oz)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_27" >
         <property name="text" >
          <string>Currency</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QWidget" native="1" name="widget_2" >
         <layout class="QVBoxLayout" name="verticalLayout_5" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="w_rate_fixed" >
            <property name="text" >
             <string>Fixed exchange rate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="w_rate_daily" >
            <property name="text" >
             <string>Daily exchange rate from ecb.eu</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <widget class="QLabel" name="label_29" >
              <property name="text" >
               <string>$1 =</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="w_currency" >
              <property name="editable" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="w_rate" />
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_28" >
         <property name="text" >
          <string>Interface</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QCheckBox" name="w_openbrowser" >
         <property name="text" >
          <string>Open a Web-Browser window when exporting to the clipboard</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QCheckBox" name="w_closeempty" >
         <property name="text" >
          <string>Close empty documents without confirmation</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" >
        <widget class="QLabel" name="label_30" >
         <property name="text" >
          <string>Document Directory</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <item>
          <widget class="QLineEdit" name="w_docdir" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="w_docdir_select" >
           <property name="icon" >
            <iconset resource="brickstore.qrc" >
             <normaloff>:/images/16x16/file_open.png</normaloff>:/images/16x16/file_open.png</iconset>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="1" >
        <spacer name="verticalSpacer_4" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="updates" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>598</width>
        <height>360</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Updates</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <item>
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>Automatically update cached data</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7" >
          <item>
           <layout class="QFormLayout" name="formLayout_6" >
            <item row="0" column="0" >
             <widget class="QLabel" name="label_13" >
              <property name="text" >
               <string>Part database, if older than</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QSpinBox" name="w_upd_database" >
              <property name="suffix" >
               <string> days</string>
              </property>
              <property name="minimum" >
               <number>1</number>
              </property>
              <property name="maximum" >
               <number>365</number>
              </property>
              <property name="value" >
               <number>7</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_10" >
              <property name="text" >
               <string>BrickLink Pictures, if older than</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QSpinBox" name="w_upd_picture" >
              <property name="suffix" >
               <string> days</string>
              </property>
              <property name="minimum" >
               <number>1</number>
              </property>
              <property name="maximum" >
               <number>365</number>
              </property>
              <property name="value" >
               <number>180</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QLabel" name="label_11" >
              <property name="text" >
               <string>BrickLink Price Guides, if older than</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" >
             <widget class="QSpinBox" name="w_upd_priceguide" >
              <property name="suffix" >
               <string> days</string>
              </property>
              <property name="minimum" >
               <number>1</number>
              </property>
              <property name="maximum" >
               <number>365</number>
              </property>
              <property name="value" >
               <number>14</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0" >
             <widget class="QLabel" name="label_12" >
              <property name="text" >
               <string>LDraw parts, if older than</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" >
             <widget class="QSpinBox" name="w_upd_ldraw" >
              <property name="suffix" >
               <string> days</string>
              </property>
              <property name="minimum" >
               <number>1</number>
              </property>
              <property name="maximum" >
               <number>365</number>
              </property>
              <property name="singleStep" >
               <number>1</number>
              </property>
              <property name="value" >
               <number>31</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_5" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>6</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_8" >
            <item row="0" column="0" >
             <spacer name="horizontalSpacer" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1" >
             <widget class="QPushButton" name="w_upd_reset" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Reset to defaults</string>
              </property>
              <property name="flat" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>117</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
      <zorder>groupBox_5</zorder>
      <zorder>w_buttons</zorder>
     </widget>
     <widget class="QWidget" name="defaults" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>598</width>
        <height>360</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Defaults</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Default values for 'Import BrickLink Set Inventory'</string>
         </property>
         <layout class="QFormLayout" name="formLayout_5" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>Item type</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="w_def_import_type" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Default values for 'Add Items'</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3" >
          <property name="fieldGrowthPolicy" >
           <enum>QFormLayout::FieldsStayAtSizeHint</enum>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Item type</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="w_def_add_type" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Condition</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QWidget" native="1" name="widget_3" >
            <layout class="QVBoxLayout" name="verticalLayout_6" >
             <property name="margin" >
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="w_def_add_cond_new" >
               <property name="text" >
                <string>New</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="w_def_add_cond_used" >
               <property name="text" >
                <string>Used</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Default values for 'Set to Price Guide'</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4" >
          <item row="0" column="0" >
           <widget class="QComboBox" name="w_def_setpg_time" />
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="w_def_setpg_price" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>39</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="network" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>598</width>
        <height>360</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Network</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>BrickLink Login</string>
         </property>
         <layout class="QFormLayout" name="formLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Username:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="w_bl_username" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="w_bl_password" >
            <property name="echoMode" >
             <enum>QLineEdit::PasswordEchoOnEdit</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="w_proxy_enable" >
         <property name="title" >
          <string>Use a Proxy for HTTP connections</string>
         </property>
         <property name="flat" >
          <bool>false</bool>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_2" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Hostname:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="w_proxy_host" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>10</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Port:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLineEdit" name="w_proxy_user" />
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_14" >
            <property name="text" >
             <string>Username:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QLineEdit" name="w_proxy_pass" >
            <property name="echoMode" >
             <enum>QLineEdit::PasswordEchoOnEdit</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="label_15" >
            <property name="text" >
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QComboBox" name="w_proxy_port" >
            <property name="editable" >
             <bool>true</bool>
            </property>
            <property name="insertPolicy" >
             <enum>QComboBox::NoInsert</enum>
            </property>
            <item>
             <property name="text" >
              <string>3128</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>8000</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>8008</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>8080</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>59</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="w_buttons" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="brickstore.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>w_buttons</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>507</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>w_buttons</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>567</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel" >
     <x>567</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
